src/main/java/emu/grasscutter/server/packet/recv/HandlerUnlockAvatarTalentReq.java

```diff
@@ -14,7 +14,7 @@ public void handle(GameSession session, byte[] header, byte[] payload) throws Ex
         UnlockAvatarTalentReq req = UnlockAvatarTalentReq.parseFrom(payload);
 
         // Unlock avatar const
-        session.getServer().getInventorySystem().unlockAvatarConstellation(session.getPlayer(), req.getAvatarGuid());
+        session.getServer().getInventorySystem().unlockNextAvatarConstellation(session.getPlayer(), req.getAvatarGuid());
```

Comment:
Might be worth actually using `req.getTalentId()` to correctly handle the very-non-vanilla case where the player can actually unlock different constellations than the lowest locked one.
![image](https://user-images.githubusercontent.com/5397662/181734734-283473af-a7c5-477b-b78a-92678c89ed3c.png)